### PR TITLE
SCRD-6919 Use SUSE colors throughout

### PR DESCRIPTION
**Pull request by GarySmith**
Update the appropriate variables to use SUSE colors everywhere.  Per the
guidance in the EOS design system pertaining to minimum contrast ratios,
a number of controls now use black text on colored background, which
is noticeably different from the standard bootstrap styling. Several
bootstrap components use colors derived programatically from the base
color (SUSE green in our case), and thus differ from the official color
variants; these include pie charts and certain disabled components.

Change-Id: I92cadda94b1f451a1d8a414efdcc9fb14b892133

### CODE REVIEW

**Review by dotNomad — CHANGES_REQUESTED** [review]
- Looks like we have adjusted the button colors, but the hover colors are not SUSE colors nor is some of the text colors. Will have to wait on guidelines for the buttons though before making changes
- `a.list-group-item, button.list-group-item` is using `color: #555`
- Panels don't appear to be using a SUSE color border, pretty minor

**Review by jeremy-moffitt — DISMISSED** [review]
Looks good to me, we should review with UX once it lands

**Comment by jeremy-moffitt** [comment]
> * Looks like we have adjusted the button colors, but the hover colors are not SUSE colors nor is some of the text colors. Will have to wait on guidelines for the buttons though before making changes
> * `a.list-group-item, button.list-group-item` is using `color: #555`
> * Panels don't appear to be using a SUSE color border, pretty minor

@dotNomad which of these do you want @GarySmith to address before merging?

**Comment by GarySmith** [comment]
I just noticed that comment. Let me provide an updated commit.

**Review by jeremy-moffitt — APPROVED** [review]
the dark menu bar is really dark... and the hover color on the yellow buttons is very orange, but overall looks ok to me

**Comment by dotNomad** [comment]
Thanks for the changes. Looks great.

**Review by dotNomad — APPROVED** [review]

**Pull request merged**
